mailet/standard/src/test/java/org/apache/james/transport/matchers/HasExceptionTest.java

```diff
@@ -47,7 +49,7 @@ public void setUp() throws Exception {
 
     @Test
     public void matchShouldReturnAddressesWhenSpecifiedExceptionHasOccurred() throws MessagingException {
-        mockedMail.setAttribute(Mail.MAILET_ERROR_ATTRIBUTE_NAME, new javax.mail.internet.AddressException());
+        mockedMail.setAttribute(new Attribute(Mail.MAILET_ERROR, AttributeValue.ofSerializable(new javax.mail.internet.AddressException())));
```

Comment:
does it have to use full class name `javax.mail.internet.AddressException`?

Reply:
@blackheaven how about this one?

Reply:
Missed it